scripts/EFI_metprocess.R

```diff
@@ -0,0 +1,68 @@
+##############################################
+#
+#       EFI Forecasting Challenge 
+#
+###############################################
+source('/projectnb/dietzelab/ahelgeso/pecan/modules/data.atmosphere/R/met.process.R')
+source('/projectnb/dietzelab/ahelgeso/pecan/modules/data.atmosphere/R/download.raw.met.module.R')
+source('/projectnb/dietzelab/ahelgeso/pecan/modules/data.atmosphere/R/GEFS_helper_functions.R')
+source('/projectnb/dietzelab/ahelgeso/pecan/modules/data.atmosphere/R/download.NOAA_GEFS.R')
```

Comment:
Right so the I think we should keep the sourced files for now, only because this was one of the ways I got around the error with download.NOAA_GEFS.R. Even though the function the set to @export  if you don't source it the convert.input section of met.process cannot find the function. This seems like an issue in it of itself and something I'm working on. But for now sourcing the scripts is the only way to get this met.process.R script to run.